keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ClassicLoadBalancerModel.kt

```diff
@@ -15,36 +15,40 @@
  */
 package com.netflix.spinnaker.keel.clouddriver.model
 
+import com.fasterxml.jackson.annotation.JsonAlias
 import com.fasterxml.jackson.annotation.JsonAnyGetter
 import com.netflix.spinnaker.keel.model.Moniker
 
 data class ClassicLoadBalancerModel(
-  val moniker: Moniker?,
-  val loadBalancerName: String,
-  val loadBalancerType: String = "classic",
-  val availabilityZones: Set<String>,
-  val vpcid: String,
-  val subnets: Set<String>,
-  val scheme: String?,
-  val listenerDescriptions: List<LoadBalancerListenerDescription>,
-  val healthCheck: LoadBalancerHealthCheck,
-  val idleTimeout: Int,
-  val securityGroups: Set<String>,
+  override val moniker: Moniker?,
+  override val loadBalancerName: String,
+  override val loadBalancerType: String = "classic",
+  override val availabilityZones: Set<String>,
+
+  @JsonAlias("vpcid", "vpcId")
```

Comment:
clouddriver currently uses `vpcid` when serving CLB's but `vpcId` for ALB and NLB's.